bin/genome-env-next

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+# if this was found via PATH as opposed to a path to executable
+THIS="$(basename "$0")"
+if test "$THIS" == "$0"
```

Comment:
You're right, this isn't doing what I thought it was.  I'll have to re-think this.

I'm trying to make sure people use the "right" version but I'm bothered by this magic/redirection.